archean/server.ini

```diff
@@ -0,0 +1,18 @@
+[server]
+server_public_name=Pelican Server
```

Comment:
Do you think we should change this to "Archean Server" in case we want to port this Egg to Pterodactyl in the future?

Reply:
> Do you think we should change this to "Archean Server" in case we want to port this Egg to Pterodactyl in the future?

I guess it doesn't really matter yeah. We could also just leave it blank because the user needs to specify a server name anyways.